examples/default.yaml

```diff
@@ -85,20 +87,21 @@ ssh:
 
 containerd:
   # Enable system-wide (aka rootful)  containerd and its dependencies (BuildKit, Stargz Snapshotter)
-  # Default: false
+  # 🟢 Builtin default: false
   system: null
   # Enable user-scoped (aka rootless) containerd and its dependencies
-  # Default: true
+  # 🟢 Builtin default: true
   user: null
 #  # Override containerd archive
-#  # Default: hard-coded URL with hard-coded digest (see the output of `limactl info | jq .defaultTemplate.containerd.archives`)
+#  # 🟢 Builtin default: hard-coded URL with hard-coded digest (see the output of `limactl info | jq .defaultTemplate.containerd.archives`)
 #  archives:
 #  - location: "~/Downloads/nerdctl-full-X.Y.Z-linux-amd64.tar.gz"
 #    arch: "x86_64"
 #    digest: "sha256:..."
```

Comment:
I wonder if this would look better as:

```yaml
containerd:
  [...]
  user: null
  # Override containerd archive
  # 🟢 Builtin default: hard-coded URL with hard-coded digest (see the output of `limactl info | jq .defaultTemplate.containerd.archives`)
  archives:
  # - location: "~/Downloads/nerdctl-full-X.Y.Z-linux-amd64.tar.gz"
  #   arch: "x86_64"
  #   digest: "sha256:..."
```

and of course the same for every empty list. This way the property name shows up more prominently, especially with syntax highlighting. And we get rid of the awkward double comments with `# # comment`.

Maybe should go into a separate PR? I can give it a try tomorrow, if you want me to.

Reply:
Thanks, I guess we can try it in a separate PR